client/jetpack-onboarding/summary-next-steps.jsx

```diff
@@ -12,9 +12,11 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import QuerySites from 'components/data/query-sites';
+import { addCalypsoEnvQueryArg } from 'jetpack-connect/utils';
 import { getEditorNewPostPath } from 'state/ui/editor/selectors';
 import { getJetpackOnboardingSettings } from 'state/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
+import { REMOTE_PATH_AUTH } from 'jetpack-connect/constants';
```

Comment:
Quoting https://github.com/Automattic/wp-calypso/pull/21755#discussion_r163195327:

> It may not be a good idea to import from these files. We may need a solution for where to keep Jetpack functions, constants, etc., which are shared across sections.